domains/transfer/Transfer.API/API.UnitTests/Converters/Base64ConverterTests.cs

```diff
@@ -28,4 +28,27 @@ public void Base64Converter_ShouldBeAbleToConvertBackAfterConversion()
         obj.PublicKey.Should().BeEquivalentTo(wde.PublicKey);
         obj.Version.Should().Be(wde.Version);
     }
+
+    [Theory]
+    [InlineData(null)]
+    [InlineData("Regular string in disguise as base64 wallet-deposit-endpoint")]
+    [InlineData("W3sibmFtZSI6ICJKb2huIn0sIHsibmFtZSI6ICJKYW5lIn1d")]
+    [InlineData("Jane Doe")]
+    public void TryConvertWalletDepositEndpoint_ShouldReturnFalse_WhenNotConvertible(string base64String)
+    {
+        var result = Base64Converter.TryConvertToWalletDepositEndpoint(base64String, out var wde);
+
+        result.Should().BeFalse();
+        wde.Should().BeNull();
+    }
+
+    [Theory]
+    [InlineData(Some.Base64EncodedWalletDepositEndpoint)]
```

Comment:
Minor thing: Just use [Fact] here as there is only one test case.

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/443/commits/af81e1579db39e8a4a95bf78dbfb95263d26410e